artemis-server/src/main/java/org/apache/activemq/artemis/core/server/ActiveMQMessageBundle.java

```diff
@@ -522,4 +522,10 @@ IllegalStateException invalidRoutingTypeUpdate(String queueName,
 
    @Message(id = 229240, value = "Connection router {0} rejected the connection", format = Message.Format.MESSAGE_FORMAT)
    ActiveMQRemoteDisconnectException connectionRejected(String connectionRouter);
+
+   @Message(id = 229241, value = "Embedded web server not found")
+   ActiveMQIllegalStateException embeddedWebServerNotFound();
+
+   @Message(id = 229242, value = "Embedded web server not restarted in {0} milliseconds", format = Message.Format.MESSAGE_FORMAT)
+   ActiveMQIllegalStateException embeddedWebServerNotRestarted(long timeout, @Cause Exception e);
```

Comment:
Using ActiveMQIllegalStateException doesnt seem quite right. Its not that the user tried to do something when it was in the wrong state, its that a result wasnt known in an allotted time, so some kind of timeout exception woudl be more appropriate.

Reply:
Fair enough. Fixed.